### PR TITLE
Added about page

### PR DESCRIPTION
**Pull request by dinojs**

### CODE REVIEW

**Comment by KashSingh1** [comment]
Mark Binghao as a TA, possibly by putting TA in brackets next to his name.

File: getting started/about.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+We propose the _ImagineThis_ tool, which takes anannotated high-fidelity prototype of a mobile user interface (UI) designed in Figma and converts it into codethat can run on both _iOS_ and _Android_. We achieved this by developing a Figma file parser, that iterates throughall layers of the Figma file, identifies components and extracts data. Then we use this information to feed intoour code generation algorithms, which output a complete React-Native project that can be compiled and runon both mobile platforms.
+
+Demo Link: https://youtu.be/dLMVbuOB5FA
+
+## 2. Sponsor
+
+Prof. Joseph Anthony Connor, NHS
+
+## 3. Supervisors
```
Comment (lukasotocerny) [inline getting started/about.md:15]: I'd add section `Teaching assistants` and reference Binghao there

File: getting started/about.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Dr. Dean Mohamedally, Dr. Emmanuel Letier, UCL
+
+## 4. Contributors
+
+### Version 2 Team
```
Comment (lukasotocerny) [inline getting started/about.md:25]: I would mention that this team are UCL students of the MSc Software Systems Engineering programme, working on this project as part of COMP0102 in 2020-2021 cohort.

File: getting started/about.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+[Denoy Hossain](https://github.com/dinojs), [Karish Singh](https://github.com/KashSingh1), [Lukas Cerny](https://github.com/lukasotocerny), [Rui Zhou](https://github.com/bbaayezi), [Zhongyuan Li](https://github.com/zhongyuanli), [Xin Deik Goh](https://github.com/DerrickGXD)
+
+### Version 1 Team
```
Comment (lukasotocerny) [inline getting started/about.md:31]: Similarly as with us, just mention that students are part of SSE programme, and summer 2020 industrial project.

**Review by lukasotocerny — CHANGES_REQUESTED** [review]

**Review by lukasotocerny — APPROVED** [review]
LGTM

**Pull request merged**